MAUI/DataForm/Editors.md

```diff
@@ -373,6 +373,36 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
+#### Numeric up-down editor
+
+The numeric up-down button will be displayed in the numeric editor, When `UpDownPlacementMode` property of `DataFormNumericItem` is set to `NumericEditorUpDownPlacementMode.Inline`. By defult, the `NumericEditorUpDownPlacementMode` is `Hidden`.
```

Comment:
The numeric up-down button will be displayed in the numeric editor when the `UpDownPlacementMode` property of `DataFormNumericItem` is set to `NumericEditorUpDownPlacementMode.Inline`. By default, the `NumericEditorUpDownPlacementMode` is `Hidden`.

Reply:
content changed